code/game/machinery/doors/firedoor.dm

```diff
@@ -116,25 +116,41 @@
 	var/mob/living/living_user = user
 
 	if (isnull(held_item))
-		if (density)
-			// This should be LMB/RMB one day
-			if (living_user.combat_mode)
-				context[SCREENTIP_CONTEXT_LMB] = "Knock"
+		if(density)
+			if(isalienadult(living_user) || issilicon(living_user))
+				context[SCREENTIP_CONTEXT_LMB] = "Open"
+				context[SCREENTIP_CONTEXT_RMB] = "Open"
```

Comment:
Why both? Is this just another case of "RMB defaults to LMB"? If so, then we should only show LMB, since I want to remove that behavior at some point.

Reply:
Same with everything else.

Reply:
I wasn't sure how to handle it so I took it the literal way. I'll get rid of it.